console/services/app_config/volume_service.py

```diff
@@ -174,35 +169,38 @@ def check_volume_name(self, service, volume_name):
         return volume_name
 
     def check_volume_path(self, service, volume_path, local_path=[]):
+        os_type = label_service.get_service_os_name(service)
+        if os_type == "windows":
+            if not runner_util.is_runner(service.image):
+                if re.match('[a-zA-Z]', volume_path[0]) and volume_path[1] == ':':
+                    if len(volume_path) == 3:
```

Comment:
What does `len(volume_path) == 3` have to do with the path of the system?

Reply:
The windows system path is usually similar to "C: /"，So the length is 3。Before judging the length, the format is judged。